src/client/proxy/http.h

```diff
@@ -242,6 +243,20 @@ class HTTPMessage : public std::enable_shared_from_this<HTTPMessage>{
 
   const unsigned int HEADERBODY_LEN = 2;
   const unsigned int REQUESTLINE_HEADERS_MIN = 1;
+ private:
+  /// @brief Checks if request is a valid jump service request
+  /// @param pos Index of m_JumpService sub-string in m_URL
```

Comment:
>m_JumpService
>m_URL

Using variable names is bad documentation style because A) it doesn't accurate describe the parameter in plain language (which is the point of documentation) and B) the names will need to be changed in any future refactoring (if the refactorer remembers to do so).

Reply:
This felt wrong when I was writing the docs as well.

Will change such instances to the high-level descriptor of the variable's use instead of it's name.